typescript/src/scss/main.scss

```diff
@@ -6,11 +12,23 @@
     margin: 20px 0;
 
     td, th {
-      text-align: right;
+      text-align: center;
       padding: 2px 5px;
-      vertical-align: top;
       border: 0;
+      justify-content: center;
     }
+    thead, .bottom{
+      background-color: rgb(136, 206, 222);
+      padding: 1.2%
+    }
+    td, .result{
```

Comment:
Really picky, but please could you add a space before the `{`? (Not sure why the linter/prettify didn't spot that.)

Reply:
Done!